js/rotate_signed_prekey_listener.js

```diff
@@ -17,7 +17,10 @@
 
     function run() {
         console.log('Rotating signed prekey...');
-        getAccountManager().rotateSignedPreKey();
+        getAccountManager().rotateSignedPreKey().catch(function() {
+            console.log('rotateSignedPrekey() failed. Trying again in five seconds');
+            setTimeout(runWhenOnline, 5000);
```

Comment:
should the communication be marked as insecure if several prekey rotations fail? Could sb game the system by blocking the rotation?

Reply:
Your prekeys are used by others to start communication with you. If you haven't updated them lately, or failed in doing so, others won't be able to start new conversations at all. No need for warnings, just try our best to make those updates happen whey are supposed to to avoid errors.

Reply:
I'll clarify: I've seen errors happen most of the time in that no-rotation case. But in theory sessions can still be created, just with fewer guarantees than a standard session.